bin/pycbc_single_template

```diff
@@ -317,7 +329,15 @@ with ctx:
         snr, corr, norm = filter.matched_filter_core(template, stilde, 
                                     psd=stilde.psd,
                                     low_frequency_cutoff=flow)
-        snr *= norm
+        snr *= norm 
+                                    
+        if opt.subtract_template:
+            stilde = subtract_template(stilde, template,
+                                snr, opt.trigger_time, opt.low_frequency_cutoff)
```

Comment:
Yes, that's right. I use the given trigger time to pick off the righ complex SNR value. This is used to scale both the distance and the phase correctly.